python/plugins/processing/algs/grass7/description/v.net.centrality.txt

```diff
@@ -1,20 +1,15 @@
 v.net.centrality
-Computes degree, centrality, betweenness, closeness and eigenvector centrality measures in the network.
+Computes degree, centrality, betweeness, closeness and eigenvector centrality measures in the network.
```

Comment:
Actually "betweenness" is correct, it's causing the spelling failure

Reply:
rats, I copied it from here https://github.com/qgis/QGIS/blob/release-2_18/python/plugins/processing/algs/grass7/description/v.net.centrality.txt#L2

Reply:
There's a good lesson there in not trusting 2.18 for **anything** 🙃

Reply:
:)